tests/test_codesigned.py

```diff
@@ -0,0 +1,97 @@
+# Copyright (C) 2014 Anaconda, Inc
+# SPDX-License-Identifier: BSD-3-Clause
+from __future__ import annotations
+
+import os
+from functools import lru_cache
+from pathlib import Path
+from shutil import which
+from subprocess import CalledProcessError, check_output, run
+
+import pytest
+
+from conda_build.utils import on_win
+
+HERE = os.path.abspath(os.path.dirname(__file__))
+REPO_ROOT = (Path(HERE) / "..").resolve().absolute()
+STUB_FOLDER = REPO_ROOT / "conda_build"
```

Comment:
You could mark these as `Final[Path]` if you wanted to.

Reply:
good point, but unless I need to touch this again before merging I likely won't add Final[Path] to these...